airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/command/DestinationConfigurationFactory.kt

```diff
@@ -15,6 +15,8 @@ interface DestinationConfigurationFactory<
     fun make(spec: I): O =
         try {
             makeWithoutExceptionHandling(spec)
+        } catch (e: ConfigErrorException) {
```

Comment:
nice! I guess we should do this for all the exception types defined here? https://github.com/airbytehq/airbyte/blob/70732cac5c232e5800ec17e0325d055938b293aa/airbyte-cdk/bulk/core/base/src/main/kotlin/io/airbyte/cdk/ConnectorErrorException.kt

(or just catch the base `ConnectorErrorException` class)

Reply:
Updated.